src/commands/server/restore.ts

```diff
@@ -34,12 +41,18 @@ export default class Restore extends Command {
     'chectl server:restore',
     '# Restore from specific backup snapshot using previos backup configuration:\n' +
     'chectl server:restore -s 585421f3',
-    '# Create and use configuration for REST backup server:\n' +
+    '# Restore from latest snapshot located in provided REST backup server:\n' +
     'chectl server:resotre -r rest:http://my-sert-server.net:4000/che-backup -p repopassword',
-    '# Create and use configuration for AWS S3 (and API compatible) backup server (bucket should be precreated):\n' +
-    'chectl server:backup -r s3:s3.amazonaws.com/bucketche -p repopassword',
-    '# Create and use configuration for SFTP backup server:\n' +
-    'chectl server:backup -r=sftp:user@my-server.net:/srv/sftp/che-data -p repopassword',
+    '# Restore from latest snapshot located in provided AWS S3 (or API compatible) backup server (bucket should be precreated):\n' +
+    'chectl server:restore -r s3:s3.amazonaws.com/bucketche -p repopassword',
+    '# Restore from latest snapshot located in provided SFTP backup server:\n' +
+    'chectl server:restore -r=sftp:user@my-server.net:/srv/sftp/che-data -p repopassword',
+    '# Rollback to previous version (if it was installed):\n' +
+    'chectl server:restore --rollback',
+    '# Restore from specific backup object:\n' +
+    'chectl server:restore --backup-cr-name=backup-object-name',
```

Comment:
Do we check if backup is successfully completed?

Reply:
I believe we need to support only the following options where all needed configuration is provided.
Admin should not guess.

1. `chectl server:restore --backup-cr-name <BACKUP_CR_NAME>`
2. `chectl server:restore VERSION+SNAPSHOT_ID+SERVER_CONFIG`
3. `chectl server:restore --rollback` ?

Reply:
I check if given backup is successful [here](https://github.com/che-incubator/chectl/pull/1680/files#diff-c5911c4982ebbd3852491fd701c846533a6f55a392fa79bedf644ae285cf06abR190).